pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
+            if (state.multipathd_running === undefined) {
```

Comment:
RIght the transition from StateNotRunning to StateRunning does not happen unless you reload the page, not even when you press "Activate" in the UI.

I'll make it so that pressing Activate will recheck the state.